doc/source/user_guide/tutorials/animate/animate_time.rst

```diff
@@ -0,0 +1,361 @@
+:ref:`ref_tutorials_import_data` tutorials section.
+
+.. jupyter-execute::
+
+    # Get a scoping of all time steps available
+    time_steps = model.metadata.time_freq_support.time_frequencies
+
+Extract the results
+-------------------
+
+Extract the results to animate. In this tutorial, you extract the displacement and stress results.
+
+.. note::
+
+    Only the |Elemental|, |Nodal|, or |Faces| locations are supported for animations.
+    |Overall| and |ElementalNodal| locations are not currently supported.
+
+
+.. jupyter-execute::
+
+    # Get the displacement fields (already on nodes) at all time steps
+    disp_fc = model.results.displacement(time_scoping=time_steps).eval()
+    print(disp_fc)
+
+.. jupyter-execute::
+
+    # Get the stress fields on nodes at all time steps
+    # Request the stress on |Nodal| location as the default |ElementalNodal| location is not supported.
+    stress_fc = model.results.stress.on_location(location=dpf.locations.nodal).on_time_scoping(time_scoping=time_steps).eval()
+    print(stress_fc)
+
+Animate the results
+-------------------
+
+Animate the results with the |animate| method.
+You can animate them on a deformed mesh (animate the color map and the mesh)
+or on a static mesh (animate the color map only).
+
+The default behavior of the |animate| method is to:
+
+- Display the norm of the data components;
+- Display data at the top layer for shells;
+- Display the deformed mesh when animating displacements;
+- Display the static mesh for other types of results;
+- Use a constant and uniform scale factor of 1.0 when deforming the mesh.
+
+You can animate any result on a deformed geometry by providing displacement results in the `deform_by` parameter.
+
+The geometry can be deformed by a |Result| object, an |Operator| (It must evaluate to a |FieldsContainer|
+of same length as the one being animated), or a |FieldsContainer| (also of same length as the one being animated).
+
+.. note::
+
+    The behavior of the |animate| method is defined by a |Workflow| it creates and feeds to an |Animator|.
+    This |Workflow| loops over a |Field| of frame indices and for each frame generates a field of norm contours
+    to render, as well as a displacement field to deform the mesh if `deform_by` is provided.
+    For more information on plots on deformed meshes see: :ref:`ref_plotting_data_on_deformed_mesh`.
+
+
+Animate the displacement results
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use |animate| with the displacement results.
+
+.. tab-set::
+
+    .. tab-item:: Deformed mesh
+
+        .. jupyter-execute::
+           :hide-output:
+
+           # Animate the displacement results in a deformed geometry
+           disp_fc.animate()
+
+        .. jupyter-execute::
+           :hide-code:
+           :hide-output:
+
+           disp_fc.animate(off_screen=True,save_as="source/user_guide/tutorials/animate/animate_disp_1.gif")
```

Comment:
Not a big fan of storing output. It forces to manually regenerate it in the future and it is prone to get outdated quickly.

In the future, it would be great to migrate this to Jupytext. I bet that using MyST can provide us with the same tab-features we are using.